plugin-server/src/cdp/services/hog-executor.service.ts

```diff
@@ -170,26 +170,22 @@ export class HogExecutorService {
             filters: HogFunctionType['filters'],
             filterGlobals: HogFunctionFilterGlobals
         ) => {
-            if (filters?.bytecode) {
-                try {
-                    const filterResults = checkHogFunctionFilters(hogFunction, filters, filterGlobals, {
-                        telemetryMatcher: this.telemetryMatcher,
-                        eventUuid: triggerGlobals.event.uuid,
-                    })
+            const filterResults = checkHogFunctionFilters({
+                hogFunction,
+                filters,
+                filterGlobals,
+                eventUuid: triggerGlobals.event.uuid,
+                enabledTelemetry: this.telemetryMatcher(hogFunction.team_id),
+            })
 
-                    // Add any generated metrics and logs to our collections
-                    metrics.push(...filterResults.metrics)
-                    logs.push(...filterResults.logs)
+            // Add any generated metrics and logs to our collections
+            metrics.push(...filterResults.metrics)
+            logs.push(...filterResults.logs)
 
-                    // Record the duration for monitoring
-                    hogFunctionFilterDuration.observe({ type: hogFunction.type }, filterResults.duration)
+            // Record the duration for monitoring
+            hogFunctionFilterDuration.observe({ type: hogFunction.type }, filterResults.duration)
 
-                    return filterResults.result
-                } catch (error) {
-                    // Already logged in the utility
```

Comment:
If we are already catching in the function then we dont want to catch here as it would mask a different issue!